services/searchengine/bleveengine/indexer/indexing_job.go

```diff
@@ -92,6 +78,10 @@ func (worker *BleveIndexerWorker) JobChannel() chan<- model.Job {
 	return worker.jobs
 }
 
+func (worker *BleveIndexerWorker) IsEnabled(cfg *model.Config) bool {
+	return true
```

Comment:
Should this be conditional on `BleveSettings.EnableIndexing`? Always true means an additional worker will always run irrespective of whether bleve is enabled or not.

Reply:
Yes, actually this was just replicating the previous existing behavior. @mgdelacroix what do you think about this?

By the way, this behavior is exactly the same in the ElasticSearch code. Should be set that to the right config setting?

Reply:
I think it would be appropriate to have it enabled depending on the config. But if it was already like this, it's not necessary to have it part of this PR.